handbook/engineering/web/frontend-platform/wildcard-component-library.md

```diff
@@ -0,0 +1,53 @@
+# Wildcard Component Library (WCL)
+
+Wildcard Component Library is a comprehensive collection of reusable, design-approved components that can be composed to quickly build out different user journeys across the web app. WCL is stored in its own [directory](https://github.com/sourcegraph/sourcegraph/tree/main/client/wildcard), owned by the [Fronted Platform Team](https://about.sourcegraph.com/handbook/engineering/web/frontend-platform) and contributed to by all Sourcegraph teammates. WCL is a part of the Wildcard Design System.
+
+## WCL x Frontend Platform Team
+- The Frontend Platform Team is the owner of the design system and Wildcard Component Library  and needs to be up-to-date with all the changes and needs of other teams
+- The Frontend Platform Team should enable other teams to make changes to the Wildcard Component Library by keeping them up-to-date (see Sharing updates to the components system), helping them plan their work, providing expertise, and reviewing the changes
+- The Frontend Platform team **shouldn't be exclusively responsible for implementing components for other teams**. Otherwise, we would create a bottleneck and the FE Platform team wouldn’t have time to focus on their team goals.
```

Comment:
Here's my suggestion:
```suggestion
- The Frontend Platform team isn't **exclusively** responsible for implementing components for other teams. Otherwise, we would create a bottleneck and the FE Platform team wouldn’t have time to focus on their other team goals.
```

Reply:
@pdubroy wdyt about it?

Reply:
lgtm. I will also add a suggestion for the next bullet point that adds more detail...will just take a couple mins.